core/code/ornaments.js

```diff
@@ -13,52 +13,120 @@
 // - Beacons (pe$TAG - $NAME) ie: 'peNIA - NIANTIC'
 // - Frackers ('peFRACK')
 // (there are 7 different colors for each of them)
+//
+// Ornament IDs are dynamic. NIANTIC might change them at any time without prior notice.
+// New ornamnent IDs found on the map will be recorded and saved to knownOrnaments from
+// which the Ornaments dialog will be filled with checked checkboxes.
+// To exclude a set of ornaments, even if they have not yet shown up on the map, the user
+// can add an entry to excludedOrnaments, which will compared (startsWith) to all known and
+// future IDs. example: "ap" to exclude all Ornaments for anomalies (ap1, ap2, ap2_v)
 
+/* global L, dialog */
 
 window.ornaments = {
 
   OVERLAY_SIZE: 60,
   OVERLAY_OPACITY: 0.6,
+  icon:[],
```

Comment:
`.icon` is an array but it's used as a object map in the code

Reply:
changed

Reply:
sidenote: avoid saying things changed when they are not pushed yet
for the reviewer perspective, nothing has changed, it's confusing

Reply:
fixed